tests/spdx/parser/rdf/test_extracted_licensing_info_parser.py

```diff
@@ -10,21 +10,51 @@
 # limitations under the License.
 import os
 
+import pytest
 from rdflib import RDF, Graph
 
 from spdx.parser.rdf.extracted_licensing_info_parser import parse_extracted_licensing_info
 from spdx.rdfschema.namespace import SPDX_NAMESPACE
 
 
-def test_parse_extracted_licensing_info():
+@pytest.mark.parametrize(
+    "parent_node_object, license_id, extracted_text, comment, license_name, cross_references",
+    [
+        (
+            SPDX_NAMESPACE.SpdxDocument,
+            "LicenseRef-1",
+            "extractedText",
+            "licenseComment",
+            "licenseName",
+            ["https://see.also"],
+        ),
+        (
+            SPDX_NAMESPACE.Package,
+            "LicenseRef-2",
+            "extractedText",
+            "licenseComment",
+            "another license",
+            ["https://see.also"],
```

Comment:
It might be helpful to give a quick explanation of the issue at hand for future developers, i.e. what's the difference between the two test cases.

Reply:
Done